tgui/packages/tgui/interfaces/Cargo.js

```diff
@@ -399,3 +483,58 @@ const CargoCart = (props, context) => {
     </Section>
   );
 };
+
+const CargoHelp = (props, context) => {
+  return (
+    <>
+      <Section title="Department Orders">
```

Comment:
I don't think we have this feature?

Reply:
we do not i thought our chefs consoles went through that for some reason